Mobile/ShopLens/ShopLensWeb/Models/User.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace ShopLensWeb
+{
+    public class User
+    {
+        public string UserId { get; set; }
+        public string Name { get; set; }
+        public DateTime Birthday { get; set; }
```

Comment:
Is there a view that calculates the user's age in our database?

Reply:
No

Reply:
I think it should be realized.

Reply:
Ok, so apparently with code first this is not so easy to achieve.
![image](https://user-images.githubusercontent.com/43152072/49339333-b62ffd80-f638-11e8-9d31-d441102210a2.png)